examples/kotlin/build.gradle

```diff
@@ -1,10 +1,12 @@
 buildscript {
+    ext.kotlin_version = '1.0.6'
+    
     repositories {
         jcenter()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:2.1.0'
-        classpath 'org.jetbrains.kotlin:kotlin-gradle-plugin:1.0.2'
+        classpath 'com.android.tools.build:gradle:2.2.3'
+        classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version"
```

Comment:
I think we should use double quotes in all Gradle scripts for strings, yes. It looks better for consistency.